compiler-plugin/src/main/kotlin/arrow/meta/quotes/ModifierList.kt

```diff
@@ -0,0 +1,50 @@
+package arrow.meta.quotes
+
+import arrow.meta.Meta
+import arrow.meta.phases.ExtensionPhase
+import org.jetbrains.kotlin.com.intellij.psi.PsiElement
+import org.jetbrains.kotlin.psi.KtAnnotationEntry
+import org.jetbrains.kotlin.psi.KtModifierList
+import org.jetbrains.kotlin.psi.psiUtil.visibilityModifier
+
+/**
+ * A [KtModifierList] [Quote] with a custom template destructuring [ModifierListScope].  See below:
+ *
+ *```kotlin:ank:silent
+ * import arrow.meta.Meta
+ * import arrow.meta.Plugin
+ * import arrow.meta.invoke
+ * import arrow.meta.quotes.Transform
+ * import arrow.meta.quotes.modifierList
+ *
+ * val Meta.reformatModifier: Plugin
+ *  get() =
+ *  "ReformatModifier" {
+ *   meta(
+ *    modifierList({ true }) { l ->
+ *     Transform.replace(
+ *      replacing = l,
+ *      newDeclaration = """ $`@annotations` $modifier value """.`modifierList`
+ *     )
+ *    }
+ *   )
+ *  }
+ *```
+ *
+ * * @param match designed to to feed in any kind of [KtModifierList] predicate returning a [Boolean]
```

Comment:
tiny nitpicky thing here - there's an extra * !

Reply:
Oh thank you to notice that, I'll remove.